addons/sale_management/static/src/js/sale_management.js

```diff
@@ -1,105 +0,0 @@
-import { rpc } from "@web/core/network/rpc";
-import publicWidget from "@web/legacy/js/public/public_widget";
-
-publicWidget.registry.SaleUpdateLineButton = publicWidget.Widget.extend({
-    selector: '.o_portal_sale_sidebar',
-    events: {
-        'click a.js_update_line_json': '_onClickOptionQuantityButton',
-        'click a.js_add_optional_products': '_onClickAddOptionalProduct',
-        'change .js_quantity': '_onChangeOptionQuantity',
-    },
-
-    /**
-     * @override
-     */
-    async start() {
-        await this._super(...arguments);
-        this.orderDetail = this.$el.find('table#sales_order_table').data();
-    },
-
-    /**
-     * Calls the route to get updated values of the line and order
-     * when the quantity of a product has changed
-     *
-     * @private
-     * @param {integer} order_id
-     * @param {Object} params
-     * @return {Deferred}
-     */
-     _callUpdateLineRoute(order_id, params) {
-        return rpc("/my/orders/" + order_id + "/update_line_dict", params);
-    },
-
-    /**
-     * Refresh the UI of the order details
-     *
-     * @private
-     * @param {Object} data: contains order html details
-     */
-    _refreshOrderUI(data){
-        window.location.reload();
-    },
```

Comment:
Maybe those methods were isolated to that they could be overridden by customizations to add parameters or handle some side effects.